x-pack/plugins/data_visualizer/common/constants.ts

```diff
@@ -40,6 +40,7 @@ export const JOB_FIELD_TYPES = {
   TEXT: 'text',
   HISTOGRAM: 'histogram',
   UNKNOWN: 'unknown',
+  VERSION: 'version',
```

Comment:
nit, `UNKNOWN` should be the last item in this list

Reply:
not for this PR, but `JOB_FIELD_TYPES` should probably be replaced with `ES_FIELD_TYPES`

Reply:
Updated here https://github.com/elastic/kibana/pull/136236/commits/7c4a14c9aae35ac086ca5594a87beb6adc093d19